src/components/show-products/index.js

```diff
@@ -117,50 +121,59 @@ export const ShowProducts = () => {
           />
         </label>
         {searchTerm && (
-          <input
+          <button
             className="close-icon"
-            onClick={() => setSearchTerm('')}
-            type="reset"
-            value="X"
             aria-label="This button clears the content of the search field."
-          />
+            onClick={() => setSearchTerm('')}
+          >
+            <FiX />
+          </button>
         )}
       </form>
 
       {list.length > 0 ? (
-        <ul>
-          {list.map(({ id, productName, lastPurchaseDate, timeFrameLabel }) => (
-            <li className="checkbox-item" key={id}>
-              <label htmlFor={id} className="checkbox-label">
-                <input
-                  type="checkbox"
-                  id={id}
-                  name={productName}
-                  onChange={(event) => handleCheckboxChange(event, id)}
-                  checked={
-                    lastPurchaseDate &&
-                    diffBetweenTodayAndDate(lastPurchaseDate.toDate()) < one_day
+        <ul className="products-list">
+          {list.map(({ id, productName, lastPurchaseDate, timeFrameLabel }) => {
+            const checked =
+              lastPurchaseDate &&
+              diffBetweenTodayAndDate(lastPurchaseDate.toDate()) < one_day;
+
+            return (
+              <li
+                className={`checkbox-item checkbox-timeFrame-${timeFrameLabel}`}
+                key={id}
+              >
+                <label
+                  htmlFor={id}
+                  className={
+                    checked ? 'checkbox-label checked-label' : 'checkbox-label'
                   }
-                  aria-label={TimeFrameLabels[timeFrameLabel]}
-                />
-                <span
-                  className={`checkmark checkbox-timeFrame-${timeFrameLabel}`}
-                ></span>
-                <span className="checkbox-name">{productName}</span>
+                >
+                  <input
+                    type="checkbox"
+                    id={id}
+                    name={productName}
+                    onChange={(event) => handleCheckboxChange(event, id)}
+                    checked={checked}
+                    aria-label={TimeFrameLabels[timeFrameLabel]}
+                  />
+                  <span className={`checkmark`}></span>
+                  {productName}
+                </label>
                 <button
                   className="button-delete"
                   aria-label={`This button deletes the product ${productName}.`}
                   onClick={() => handleDeleteProduct(id, productName)}
                 >
-                  Delete
+                  <FiTrash2 />
                 </button>
-              </label>
-            </li>
-          ))}
+              </li>
+            );
+          })}
         </ul>
       ) : (
         <p className="empty-shopping-list">{`There aren't products that match with '${searchTerm} '`}</p>
```

Comment:
I find a small detail when you are trying to find some product with your filter.

![image](https://user-images.githubusercontent.com/8689897/142733130-e008946a-7bba-41bb-94c0-d946880279ba.png)

I think you can add some additional control in your form search (maybe your input) like `maxLength`. What do you think about it?

Reply:
Probably, some padding would work   ^^

Reply:
Solved! Thanks for the suggestions! @alxmcr  and @yanurd 

![localhost_3000_list(Moto G4)](https://user-images.githubusercontent.com/32148199/142742929-78472d03-afba-4925-97ba-d2f09dafdb43.png)